orc8r/cloud/go/obsidian/tests/unit_test_utils.go

```diff
@@ -91,7 +91,9 @@ func RunUnitTest(t *testing.T, e *echo.Echo, test Test) {
 			expectedBytes, err := test.ExpectedResult.MarshalBinary()
 			if assert.NoError(t, err) {
 				// Convert to string for more readable assert failure messages
-				assert.Equal(t, string(expectedBytes), string(recorder.Body.Bytes()))
+				// Add newline to expectedBytes due to Json.Marshal appending
+				// a newline when marshaling
+				assert.Equal(t, string(expectedBytes)+"\n", string(recorder.Body.Bytes()))
```

Comment:
Unclear why this popped up with the given go.mod changes since there was no update to `encoding/json` that I can see. Either way this seems to fix all the related tests.